src/composables/useLeaflet.ts

```diff
@@ -123,6 +126,23 @@ export const useLeaflet = (
             }).addTo(leafletMap);
         }
 
+        if (selectedAdminLevel.value === 2) {
+            const selectedAdmin1Features = admin1Geojson.value[regionId];
+            selectedAdmin1OutlineLayer.value = selectedAdmin1Features?.map((f) => {
+                if (!f?.geometry?.type || !f?.geometry?.coordinates) return null;
+                const latLngs = GeoJSON.coordsToLatLngs(
+                    f.geometry.coordinates,
+                    f.geometry.type === "MultiPolygon" ? 2 : 1
+                );
+                return polyline(latLngs, {
+                    color: "grey",
+                    weight: 0.4,
+                    opacity: 0.75,
+                    className: "admin-1-outline"
+                }).addTo(leafletMap);
```

Comment:
It's starting to feel like maybe it would nice to pull these hardcoded styles out into a separate place, maybe the map utils?